assets/js/src/core/modules/element/dynamic-types/defintinitions/objects/data-related/types/abstract/dynamic-type-object-data-abstract-numeric.tsx

```diff
@@ -45,25 +45,27 @@ const fixUnsigned = (unsigned: boolean, value: number | null, max: boolean): num
 }
 
 export abstract class DynamicTypeObjectDataAbstractNumeric extends DynamicTypeObjectDataAbstract {
-  getObjectDataComponent (props: AbstractNumericObjectDataDefinition): React.ReactElement<AbstractObjectDataDefinition> {
-    return (
-      <InputNumber
-        { ...this.getObjectDataComponentProps(props) }
-      />
-    )
-  }
-
   getObjectDataComponentProps (props: AbstractNumericObjectDataDefinition): IInputNumberProps {
     return {
       inherited: props.inherited,
       disabled: props.noteditable === true,
       max: fixUnsigned(props.unsigned === true, props.maxValue, true) ?? undefined,
       min: fixUnsigned(props.unsigned === true, props.minValue, false) ?? undefined,
+      defaultValue: props.value,
```

Comment:
We need to solve this differently - we cannot use defaultValue as this will break the data object editor handling. If there is no other way then changing the defaultValue we would need to do this in the `getVersionObjectDataComponent`